core/storage/suggestion/gae_models.py

```diff
@@ -1292,3 +1292,678 @@ def export_data(
                     date.isoformat() for date in model.contribution_dates]
             }
         return user_data
+
+
+class TranslationReviewStatsModel(base_models.BaseModel):
+    """Records the contributor dashboard translation review stats. There
+    is one instance of this model per (language_code, reviewer_user_id,
+    topic_id) tuple. See related design doc for more details:
```

Comment:
I think you can drop the last sentence. All info here should be self-contained.

Reply:
Done.